hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/HBaseRpcController.java

```diff
@@ -37,7 +39,8 @@
  * RegionInfo we're making the call against if relevant (useful adding info to exceptions and logs).
  * Used by client and server ipc'ing.
  */
-@InterfaceAudience.Private
+@InterfaceAudience.LimitedPrivate({HBaseInterfaceAudience.COPROC, HBaseInterfaceAudience.PHOENIX})
```

Comment:
@apurtell - we may want to also add HBaseInterfaceAudience.REPLICATION to the LimitedPrivate annotation. @bharathv and I recently tried to make a custom replication sink implementing the replication API (which I assume counts as Public?), but found we had to downcast RpcController to HBaseRpcController to get at the actual replicated Cells.

Reply:
@gjacoby126 Updated